frontend/util.lua

```diff
@@ -699,7 +699,8 @@ function util.findFiles(dir, cb)
         if not ok then return end
         for f in iter, dir_obj do
             local path = current.."/"..f
-            local attr = lfs.attributes(path)
+            -- lfs can return nil here, as it will follow symlinks!
+            local attr = lfs.attributes(path) or {}
```

Comment:
Does this problem apply elsewhere?

Reply:
At a quick glance, most lfs callers ask for a specific attribute, meaning there's no "trying to index a table that doesn't exist" issue down the line. We can't do that here because the callback expects the full attributes table.

The few that do something similar either do this or have nil guards in place.